integration_tests/tests/transaction-view-athena-tests.ts

```diff
@@ -31,12 +34,16 @@ describe("\nExecute athena transaction curated query to retrieve price \n", () =
       eventTime: string;
     }) => {
       const expectedPrice = (numberOfTestEvents * unitPrice).toFixed(4);
-      await generateTransactionEventsViaFilterLambda(
-        eventTime,
-        numberOfTestEvents,
-        eventName
-      );
-
+      for (let i = 0; i < numberOfTestEvents; i++) {
+        const createEventPayload = await generateTestEvent({
```

Comment:
As mentioned previously, I think the variable name can just be eventPayload.

Reply:
done